### ISSUE TITLE
ImportError: librrd.so.4: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by blodone** [commx/python-rrdtool#20]
On Debian Stretch

---> 1 import rrdtool
ImportError: librrd.so.4: cannot open shared object file: No such file or directory

locate librrd:
/usr/lib/x86_64-linux-gnu/librrd.a
/usr/lib/x86_64-linux-gnu/librrd.so
/usr/lib/x86_64-linux-gnu/librrd.so.8
/usr/lib/x86_64-linux-gnu/librrd.so.8.0.0
/usr/lib/x86_64-linux-gnu/pkgconfig/librrd.pc

**Comment by commx** [comment]
Hi,

I just did a fresh minimal install of Debian 9.1 ("stretch"), installed the `librrd-dev` package, along with `python3-pip` and `python-pip` (for both Python installed versions). Installing the rrdtool package works just fine and I can import the rrdtool package without any errors:

```
# apt install -y librrd-dev python-pip python3-pip
# pip install rrdtool
```

```
>>> import rrdtool
>>> rrdtool
<module 'rrdtool' from '/usr/local/lib/python2.7/dist-packages/rrdtool.so'>
```

Did you do a Debian upgrade recently and haven't re-built the python-rrdtool module with an updated librrd version? Debian 9 seems to ship rrdtool 1.6.x whereas the previous version shipped rrdtool 1.4.x. If so, try uninstalling python-rrdtool and re-install it by omitting possible cached versions:

```
# pip uninstall rrdtool -y
# pip install --no-cache-dir rrdtool
```

This will most likely try to build the module from source rather than using a pre-built wheel.

**Comment by blodone** [comment]
Hi, thanks for your respond.

I did:
`pip uninstall rrdtool
pip install rrdtool`

now it works!
